build.gradle

```
@@ -44,27 +44,27 @@ modifyPom {
     project {
         name 'Sentry Android Gradle Plugin'
         description 'Sentry Android Gradle Plugin'
-        url 'https://github.com/getsentry/sentry-java'
+        url 'https://github.com/getsentry/sentry-android-gradle-plugin'
 
         scm {
-            url 'https://github.com/getsentry/sentry-java'
-            connection 'scm:https://github.com/getsentry/sentry-java.git'
-            developerConnection 'scm:git@github.com:getsentry/sentry-java.git'
+            url 'https://github.com/getsentry/sentry-android-gradle-plugin'
+            connection 'scm:https://github.com/getsentry/sentry-android-gradle-plugin.git'
+            developerConnection 'scm:git@github.com:getsentry/sentry-android-gradle-plugin.git'
         }
 
         licenses {
             license {
-                name 'BSD'
-                url 'https://github.com/getsentry/sentry-java/blob/master/LICENSE'
+                name 'MIT'
+                url 'https://github.com/getsentry/sentry-android-gradle-plugin/blob/master/LICENSE'
                 distribution 'repo'
             }
         }
 
         developers {
             developer {
-                id 'bretthoerner'
-                name 'Brett Hoerner'
-                email 'brett@sentry.io'
+                id 'sentry-io'
+                name 'Sentry Team and Contributors'
+                email 'oss@sentry.io'
```

```suggestion
                email 'accounts@sentry.io'
```

or we change sentry-java too